tests/integration/disambiguation/test_disambiguation_api.py

```diff
@@ -33,15 +34,59 @@
     save_publications,
     save_signatures_and_clusters,
     train_and_save_ethnicity_model,
+    train_and_save_distance_estimator
 )
 from inspirehep.modules.disambiguation.core.ml.models import EthnicityEstimator
+from inspirehep.modules.disambiguation.core.ml.models import DistanceEstimator
 
-TRAINING_DATA = '''\
+ETHNICITY_TRAINING_DATA = '''\
 RACE,NAMELAST,NAMEFRST
 1,EASTWOOD,CLINT
 5,MIFUNE,TOSHIRO
 '''
 
+DISTANCE_ESTIMATOR_SIGNATURES_DATA = [
+    {
+        'signature_uuid': '1',
+        'publication_id': 1,
+        'author_name': 'Litov, Leander',
+        'author_id': 1,
+        'author_affiliation': 'Random Affiliation',
+        'signature_block': 'LATAVl'
+    },
+    {
+        'signature_uuid': '2',
+        'publication_id': 2,
+        'author_name': 'Litov, Leander',
+        'author_id': 2,
+        'author_affiliation': 'Another Made Up Affiliation',
+        'signature_block': 'LATAVl'
+    }
+]
```

Comment:
It won't hurt to use real data fixtures. In particular what worries me is that what is a `signature_uuid` here is simply a string, which is not a problem now, but might be.

For the ethnicity data I used fake data because we don't want to leak that data set which isn't ours, but for this one there's no problem.